### ISSUE TITLE
Disable linter fails after comment on previous line

### ISSUE THREAD

**Issue by bartveneman** [brigade/scss-lint#373]
I use a commenting system with numbers, so I don't have to put all of them inline, which can become really sloppy:

````scss
// 1. Some comment about my background
.foo {
  background: 0; // [1]
  border-radius: 0;
}
````

When using an inline scss-lint:disable comment, it breaks when it comes right after a line with this comment, like so:

````scss
// 1. Some comment about my background
.foo {
  background: none; // [1]
  // scss-lint:disable ImportantRule
  border-radius: 0 !important;
  // scss-lint:enable ImportantRule
}
````
It now reports an error on line 5 to not use `!important`. Removing the `// [1]` comment makes the inline disable statement work like it should. The same goes for when I wrap the whole selector in the toggling comments.

**Comment by sds** [comment]
@bartveneman thanks for the report. This is a problem with the `sass` parser. The previous comment (`// [1]`) is treated as the beginning of a multi-line comment, so the sass parser sees the following comment as just a continuation.

This is unexpected, and breaks some fundamental assumptions our control comment processing code makes.

I won't have time to look into this for a while, so my suggestion is to avoid your commenting scheme, or submit a PR that addresses the issue. Thanks!

**Comment by ixley** [comment]
+1 stumbled onto this bug as well.

**Comment by sds** [comment]
Fixed by #539.
